collection-scripts/gather

```diff
@@ -33,11 +33,13 @@ resources+=(imagecontentsourcepolicies.operator.openshift.io)
 resources+=(networks.operator.openshift.io)
 
 # Run the Collection of Resources using inspect
-# running accross all-namespaces for the few "Autoscaler" resouces.
 for resource in "${resources[@]}"; do
     oc adm inspect --dest-dir must-gather ${resource}
 done
 
+# Gather optional operator resources from all namespaces
+oc adm inspect --dest-dir must-gather -A olm
```

Comment:
`-A` is limited to what `oc` versions (we need to make sure we don't back-port this past that point (and this introduce errors/failures in the collection process). 

cc: @mfojtik